test/Pester/PlatyPs.Tests.ps1

```diff
@@ -379,6 +385,54 @@ Describe 'New-MarkdownHelp' {
 
         }
     }
+	
+	Context 'Full Type Name' {
+        function global:Get-Alpha
+        {
+            param(
+                [Switch]
+                [Parameter(Position=1)]
+                $WhatIf,
+                [string]
+                [Parameter(Position=2)]
+                $CCC
+            )
+        }
+
+        It 'use full type name when specified' {
+            $expectedParameters = normalizeEnds @'
+Type: System.String
+Type: System.Management.Automation.SwitchParameter
+
+'@ 
+            $expectedSyntax = normalizeEnds @'
+Get-Alpha [[-WhatIf] <System.Management.Automation.SwitchParameter>] [[-CCC] <System.String>]
+
+'@
+
+            $files = New-MarkdownHelp -Command Get-Alpha -OutputFolder TestDrive:\alpha -Force -AlphabeticParamsOrder -UseFullTypeName
+            ($files | Measure-Object).Count | Should Be 1
+            Get-Content $files | Where-Object {$_.StartsWith('Type: ')} | Out-String | Should Be $expectedParameters
+            Get-Content $files | Where-Object {$_.StartsWith('Get-Alpha')} | Out-String | Should Be $expectedSyntax
+        }
+
+        It 'not use full type name when specified' {
+            $expectedParameters = normalizeEnds @'
+Type: String
+Type: SwitchParameter
+
+'@ 
```

Comment:
I don't understand why this extra blank line is needed. Maybe if you remove it, you will be able to avoid `normalizeEnds` ?